overlord/hookstate/hookmgr.go

```diff
@@ -142,6 +147,18 @@ func (m *HookManager) Stop() {
 	m.runner.Stop()
 }
 
+func (m *HookManager) hijacked(hookName, snapName string) HookFunc {
+	return m.hijackedMap[fmt.Sprintf("%s:%s", hookName, snapName)]
```

Comment:
The key for this map might also be a struct:
```
type hijackKey struct { hook, snap string }
```
This way we don't need to sprintf every time:
```
return m.hijackMap[hijackKey{hookName, snapName}]
```